### DISCUSSION TITLE
Discussions is open for use as a temporary measure whilst the BBS is down.

### DISCUSSION THREAD

**Discussion by mikedlr** [eegeeZA/spamgourmet-clone#72] [Announcements]
These discussions are now open for use. I will probably close them down when the BBS gets back up and working. That should be a short time, but in the meantime the discussions here should be useful for sharing information even though they don't have the huge history which is so valuable.

**Comment by inaun** [comment]
First, I want to highlight how much I appreciate this service -- and your carrying it on!

It seems like we are not going to win the battle for getting spamgourmoet off the "disposable e-mail address" block lists. What do you think about having some new domain names that are branded as an anti-spam service without any clear connection to Spamgourmet, so the F*****g marketing people can't easily figure out what is happening? I'd be willing to help with such an endeavor. The front page could talk about blocking spam, tracking who is selling your e-mail, and regaining control instead of highlighting the temporary nature of the address keywords.

The most difficult part would probably be determining how to hide all ties back to Spamgourmet, so the new domains don't get picked up by the f*****g marketing companies who seem to thing THEY should own our e-mail.

**Reply by SysKoll** [reply]
Sadly, it is trivial to identify a new domain name linked to sg. All you have to do is fetch the MX record from the DNS, and compare the returned address with the one for spamgourmet.com. So if someone decides to block sg, a new domain name is not going to cut it. You also need a separate server with its own IP.

**Reply by sigfridvonshrink** [reply]
this might be managed by adding email addresses to the server and pointing the new MX records to them. But I doubt it would be a long time solution - a marketer might simply create an account and then watch from which domains the emails come.
